extensions/wikia/RTE/js/plugins/placeholder/plugin.js

```diff
@@ -403,9 +362,10 @@ CKEDITOR.plugins.add('rte-placeholder',
 				//RTE.log([x, y]);
 
 				// calculate editarea size
-				var editarea = $('#cke_contents_wpTextbox1');
-				var maxX = parseInt(editarea.offset().left) + editarea.width();
-				var maxY = parseInt(editarea.offset().top) + editarea.height();
+				var editArea = document.getElementById('cke_1_contents'),
```

Comment:
getElementById can return undefined, should we guard against it?

Reply:
@hakubo if this will return undefined then it means that editor is not loaded properly and functionality is broken. Do you think that complicating this code is necessary if editor is broken anyway in case of undefined?